### ISSUE TITLE
Integration with Skupper

### ISSUE THREAD

**Issue by rigazilla** [infinispan/infinispan-operator#385]
Skupper (https://skupper.io/) is a k8s component that provides intercluster communication.
Investigate if it can be useful to semplify cross site replication configuration.

**Comment by dmvolod** [comment]
I think we can try to utilize both solutions, but first step `install and configure Skupper or Submariner` will be so huge
However next steps will be more easy

**Comment by pruivo** [comment]
at least, the submariner has an operator: https://operatorhub.io/operator/submariner

not sure how good it is neither what it does xD

**Comment by dmvolod** [comment]
> not sure how good it is neither what it does xD

Yeah, it utilize `sigs.k8s.io/mcs-api` and easy to expose cross-site services

**Comment by dmvolod** [comment]
But seems to need access to the remote cluster Kubernetes API. In this case there is no value, except granting access on the k8s cluster level, not individual Infinispan 
https://submariner.io/getting-started/architecture/
